src/components/OrganizationsDropdown.tsx

```diff
@@ -60,7 +60,7 @@ export default function OrganizationsDropdown(): JSX.Element {
         onSuccess={(organization) => void handleSuccess(organization)}
       />
       <PopoverMenu
-        anchor={<p style={{ fontSize: '16px' }}>{selectedOrganization.name}</p>}
+        anchor={<p style={{ fontSize: '16px' }}>{selectedOrganization?.name}</p>}
```

Comment:
I was torn on whether to turn this kind of thing into a higher-level guard clause vs. a null-safe reference.